### PR TITLE
update README.md

### PR DESCRIPTION
**Pull request by Mudassar045**
Just a few minor changes

### CODE REVIEW

**Comment by ryangjchandler** [comment]
I'm not sure if all of these changes are valid, grammatically.

File: README.md [hunk 1]
```diff
@@ -181,15 +181,15 @@ You can also mix-in multiple data objects using object destructuring:
...
+`x-bind` sets the value of an attribute to the result of a JavaScript expression. The expression has access to all the keys of the component's data object, and will update every-time its data is updated.
 
 > Note: attribute bindings ONLY update when their dependencies update. The framework is smart enough to observe data changes and detect which bindings care about them.
 
 **`x-bind` for class attributes**
 
 `x-bind` behaves a little differently when binding to the `class` attribute.
 
-For classes, you pass in an object who's keys are class names, and values are boolean expressions to determine if those class names are applied or not.
+For classes, you pass in an object which keys are class names, and values are boolean expressions to determine if those class names are applied or not.
```
Comment (lancebutler2) [inline README.md]:
```suggestion
For classes, you pass in an object whose keys are class names, and values are boolean expressions to determine if those class names are applied or not.
```

Comment (Mudassar045) [reply README.md]: You said that `I'm not sure if all of these changes are valid, grammatically.` But dude, I'm sure about.

Comment (lancebutler2) [reply README.md]: I didn't say that.

And what are you sure about?

Comment (Mudassar045) [reply README.md]: **I didn't say**: https://github.com/alpinejs/alpine/pull/58#issuecomment-572192579
**And what are you sure about**: I'm sure about the grammar"

In short, all the changes made in PR are not related to grammar, but a few, and among them, your pointing one. For this particular case, `that`, `which` and `who`, can be used but I used `which`.

Comment (lancebutler2) [reply README.md]: I did not make that comment. That is someone else.

Using `that`, `which` or `who` would all be incorrect. Each one of those words would make this an incorrect sentence. You're correct in suggesting `who's` be changed.  It should, however, be updated to `whose`.

All of your other grammar-related suggested changes are spot-on!

File: README.md [hunk 2]
```diff
@@ -139,9 +139,9 @@ You can extract data (and behavior) into reusable functions:
     function dropdown() {
         return {
             show: false,
-            open() { this.show = true },
-            close() { this.show = false },
-            isOpen() { return this.show === true },
+            open(): { this.show = true },
+            close(): { this.show = false },
+            isOpen(): { return this.show === true },
```
Comment (lancebutler2) [inline README.md]: These 3 suggestions are syntax errors. It was already written correctly.

File: README.md (hunk 1 above)
Comment (Mudassar045) [reply README.md]: Apologies

**Comment by calebporzio** [comment]
Closing this because it's not entirely accurate - thanks for the effort though.

**Pull request closed without merging**